src/environment.cpp

```diff
@@ -623,7 +622,7 @@ void ServerEnvironment::savePlayer(RemotePlayer *player)
 	player->save(players_path, m_gamedef);
 }
 
-RemotePlayer *ServerEnvironment::loadPlayer(const std::string &playername)
+RemotePlayer *ServerEnvironment::loadPlayer(const std::string &playername, PlayerSAO *sao)
```

Comment:
This is not possible at this moment because playerSAO is not attached to remotePlayer it's done later in the emerge process